<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKChartsXYSerialWidget</class>
 <widget class="QWidget" name="SAKChartsXYSerialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>波形图</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QPushButton" name="deletePushButton">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="editPushButton">
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="chartSettingsPushButton">
     <property name="text">
      <string>图表设置</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="clearPushButton">
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QWidget" name="chartViewerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="addPushButton">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="pauseCheckBox">
     <property name="text">
      <string>暂停输入</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
